tests/functional/oauth_query_param_validation.js

```diff
@@ -55,27 +63,19 @@ registerSuite('oauth query parameter validation', {
     return this.remote
       .then(clearBrowserState({
         contentServer: true
-      }));
+      }))
+      .then(openFxaFromRp('signup'))
```

Comment:
Thanks for moving this to `beforeEach`, when I had a quick look at this module I realized how bad of an idea [using the first test to set up TRUSTED_CLIENT_ID and UNTRUSTED_CLIENT_ID](https://github.com/mozilla/fxa-content-server/pull/6214/files#diff-ea9f8f7971cb8bbbbb7351fabb68ca7fL61) was in the first place. I should have never done that.

Reply:
@vladikoff 👍